### DISCUSSION TITLE
[Question][Hw4] __dbz_sanitizer__ input type error

### DISCUSSION THREAD

**Discussion by polish23** [prosyslab-classroom/cs348-information-security#553] [Q&A]
When I run the code like below, there's an error like this.
![problem](https://user-images.githubusercontent.com/79696492/117912259-266aff00-b31a-11eb-95a6-dd63350868a7.JPG)

I think division input for __dbz_sanitizer__ must be int, int, int.
However, there's an double input in json_parser.c

![KakaoTalk_20210512_115643369](https://user-images.githubusercontent.com/79696492/117912364-5adebb00-b31a-11eb-92ab-fae5555f8bcf.png)

How can I solve this problem?

Thanks.

**Comment by KihongHeo** [comment]
It seems that your sanitizer instruments also `FDiv`. You can just remove that part from your sanitizer.

**Comment by polish23** [comment]
Thanks!
